dev-libs/Ice/Ice-3.6.5.ebuild

```diff
@@ -0,0 +1,336 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_6,3_7} )
```

Comment:
Please test python3_8 as well if possible. While changing it you could also change the whole line to this: `PYTHON_COMPAT=( python{2_7,3_{6,7,8}} )`

Reply:
as i said in the initial post, my aim was to provide that in a revbump once this hits stable. that is *if* the patch is even backportable from 3.7, as the revbump needs to happen for ruby 2.6 anyway i thought it might make sense this way.